charts/application/values.yaml

```
@@ -12,6 +12,8 @@ container:
   # annotations to be added to the pod template
   annotations:
     "cluster-autoscaler.kubernetes.io/safe-to-evict": "true"
+  # command line given to the container; []one string
+  # command: []
```

can't this be uncommented?
```suggestion
  command: []
```

---

yes, you're right. will change once the other PR is merged (to avoid unnecessary workflow runs)